app/components/StudioFooter.vue.ts

```diff
@@ -35,9 +36,16 @@ export default class StudioFooterComponent extends Vue {
   @Prop() locked: boolean;
 
   metricsShown = false;
+  recordingTime = '';
 
   mounted() {
     this.confirmYoutubeEnabled();
+
+    // update recording time
+    setInterval(() => {
+      if (!this.streamingService.isRecording) return;
+      this.recordingTime = this.streamingService.formattedDurationInCurrentRecordingState;
+    }, 1000);
```

Comment:
You should clear this interval in the `destroyed` hook.  I know the footer is unlikely to ever unmount, but it's good practice to ensure we never leak interval handlers.